cistem/protocols/protocol_tomo_resample.py

```diff
@@ -0,0 +1,184 @@
+# **************************************************************************
+# *
+# * Authors:     Ricardo D. Righetto (ricardo.righetto@unibas.ch)
+# *
+# * University of Basel
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+
+'''
+A protocol to resample tomograms by Fourier cropping/padding using cisTEM.
+'''
+from cistem import Plugin
+from pwem.protocols import EMProtocol
+from pyworkflow import BETA
+from pyworkflow.protocol import PointerParam, IntParam
+from pyworkflow.utils import *
+from tomo.objects import SetOfTomograms, Tomogram
+from pwem.convert.headers import Ccp4Header
+
+OUTPUT_TOMO_NAME = 'resampledTomos'
+OUTPUT_DIR = 'extra/'
+
+class ProtTomoResample(EMProtocol):
+    '''
+    Resample tomograms by Fourier cropping/padding using cisTEM. This is equivalent to binning/unbinning operations but free of aliasing artifacts.
+
+    More info:
+        https://cistem.org
+    '''
+
+    _label = 'resample tomogram'
+    _possibleOutputs = {OUTPUT_TOMO_NAME: SetOfTomograms}
+    _devStatus = BETA
+
+    tomoList = []
+
+    def __init__(self, **args):
+        EMProtocol.__init__(self, **args)
```

Comment:
I suppose no point re-defining __init__ without adding any extra functionality. Parent class init will be used anyway